### PR TITLE
Bump versions for individual parents and add LWT tests

### LINKED ISSUE #1139: Version bump for file format and LWT for `Individual.parents`

### PR DESCRIPTION
**Pull request by benjeffery**
Fixes #1139

### CODE REVIEW

File: c/CHANGELOG.rst [hunk 1]
```diff
@@ -4,8 +4,13 @@
 
 **Features**
 
+- Add ``parents`` to the individual table to enable recording of pedigrees
+  (:user:`ivan-krukov`, :issue:`852`, :pr:`1125`, :pr:`866`, :pr:`1153`).
```
Comment (jeromekelleher) [inline c/CHANGELOG.rst:8]: I think we need :user:`benjeffery` in there too!

File: python/lwt_interface/CHANGELOG.rst [hunk 2]
```diff
@@ -1,3 +1,9 @@
+--------------------
+[0.1.2] - 2021-02-01
```
Comment (jeromekelleher) [inline python/lwt_interface/CHANGELOG.rst:2]: 0.1.3?

**Review by jeromekelleher — APPROVED** [review]
LGTM, one nit.

File: c/CHANGELOG.rst (hunk 1 above)
Comment (benjeffery) [reply c/CHANGELOG.rst]: Done

File: python/lwt_interface/CHANGELOG.rst (hunk 2 above)
Comment (benjeffery) [reply python/lwt_interface/CHANGELOG.rst]: Done

**Pull request merged**